lightning/src/ln/interactivetxs.rs

```diff
@@ -168,6 +171,61 @@ pub(crate) struct ConstructedTransaction {
 	holder_sends_tx_signatures_first: bool,
 }
 
+impl Writeable for ConstructedTransaction {
+	fn write<W: Writer>(&self, writer: &mut W) -> Result<(), io::Error> {
+		let lock_time = self.lock_time.to_consensus_u32();
```

Comment:
Should we implement `Readable` / `Writeable` for `LockTime` so we can use one macro to implement these?